images/19.0.0-php8.2/Dockerfile

```diff
@@ -40,26 +40,26 @@ ENV PHP_INI_POST_MAX_SIZE 8M
 ENV PHP_INI_ALLOW_URL_FOPEN 0
 
 RUN sed -i \
-  -e 's/^\(ServerSignature On\)$/#\1/g' \
-  -e 's/^#\(ServerSignature Off\)$/\1/g' \
-  -e 's/^\(ServerTokens\) OS$/\1 Prod/g' \
-  /etc/apache2/conf-available/security.conf
+    -e 's/^\(ServerSignature On\)$/#\1/g' \
+    -e 's/^#\(ServerSignature Off\)$/\1/g' \
+    -e 's/^\(ServerTokens\) OS$/\1 Prod/g' \
+    /etc/apache2/conf-available/security.conf
 
 RUN apt-get update -y \
     && apt-get dist-upgrade -y \
     && apt-get install -y --no-install-recommends \
-        libc-client-dev \
-        libfreetype6-dev \
-        libjpeg62-turbo-dev \
-        libkrb5-dev \
-        libldap2-dev \
-        libpng-dev \
-        libpq-dev \
-        libxml2-dev \
-        libzip-dev \
-        default-mysql-client \
-        postgresql-client \
-        cron \
+    libc-client-dev \
+    libfreetype6-dev \
+    libjpeg62-turbo-dev \
+    libkrb5-dev \
+    libldap2-dev \
+    libpng-dev \
+    libpq-dev \
+    libxml2-dev \
+    libzip-dev \
+    default-mysql-client \
+    postgresql-client \
+    cron \
```

Comment:
Why add/remove spaces here ?

Reply:
This is not voluntary but the default formatting of vscode. I will revert this change.